File: ClientApp/src/app/models/organizacion.ts
import { IBase } from "./base";
import { IUsuario } from "./usuario";
import { IUsuarioOrganizacion } from "./usuarioOrganizacion";

export interface IOrganizacion extends IBase{
    nombre: string
    descripcion: string
    contactoId: number
    contacto: IUsuario
    usuarioOrganizacion: IUsuarioOrganizacion[]
}

File: Models/Organizacion.cs
using System.Collections.Generic;

namespace Donaciones.Models
{
    public class Organizacion:Base
    {
        public string Nombre { get; set; }
        public string Descripcion { get; set; }
        public int? ContactoId { get; set; }
        public Usuario Contacto { get; set; }
        public IEnumerable<UsuarioOrganizacion> UsuarioOrganizaciones { get; set; }
        public bool EstaActivo { get; set; } = true;
    }
}
